apps/assistants/tables.py

```diff
@@ -1,29 +1,39 @@
 from django.conf import settings
+from django.utils.safestring import mark_safe
 from django_tables2 import columns, tables
 
 from apps.assistants.models import OpenAiAssistant
 from apps.generics import actions
 
 
+def get_assistant_row_attrs():
+    def _get_redirect_url(record):
+        return record.get_absolute_url() if record.working_version is None else ""
+
+    return {
+        **settings.DJANGO_TABLES2_ROW_ATTRS,
+        "data-redirect-url": _get_redirect_url,
+    }
+
+
 class OpenAiAssistantTable(tables.Table):
     name = columns.Column(
-        linkify=True,
-        attrs={
-            "a": {"class": "link"},
-        },
+        linkify=False,
         orderable=True,
     )
     actions = actions.ActionsColumn(
         actions=[
             actions.edit_action(
                 "assistants:edit",
                 required_permissions=["assistants.change_openaiassistant"],
+                display_condition=lambda request, record: record.working_version is None,
```

Comment:
avoid querying the working version:

```suggestion
                display_condition=lambda request, record: record.working_version_id is None,
```